src/Components/Inputs/Documentation/Inputs.stories.tsx

```diff
@@ -1,5 +1,5 @@
 // Libraries
-import React, {RefObject, createRef} from 'react'
+import React, {RefObject, createRef, useState} from 'react'
```

Comment:
Improt the story `useState` from storybook instead of react. `import {useState} from '@storybook/addons'`